_config.yml

```diff
@@ -1,16 +1,18 @@
-title: "Jekyll Plugins • Community-built plugins to level up your Jekyll experience."
+title: "Jekyll directory • Community-built themes and plugins to level up your Jekyll experience."
 repository: https://github.com/jekyll/plugins
 url: http://jekyll.github.io
-baseurl: /plugins
+baseurl: /directory
```

Comment:
I would let GitHub Pages do this for you automatically: https://github.com/jekyll/github-metadata/blob/8906f2b9c890f0aafef96423c7cd7e5047f7dae4/lib/jekyll-github-metadata/ghp_metadata_generator.rb#L36-L45